src/pages/settings/Wallet/WalletSection.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Section from '../../../components/Section';
+
+const propTypes = {
+    icon: PropTypes.func.isRequired,
+    children: PropTypes.node.isRequired,
+    subtitle: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+};
+
+function WalletSection(props) {
+    const {icon, children, subtitle, title} = props;
+    return (
+        <Section
+            icon={icon}
+            subtitle={subtitle}
+            title={title}
+        >
+            {children}
+        </Section>
+    );
+}
+
+WalletSection.propTypes = propTypes;
```

Comment:
add the `displayName`

Reply:
and `defaultProps`

Reply:
@thiagobrez There are no optional props in this component, so it's not necessary 😄